packages/tx/src/7702/tx.ts

```diff
@@ -117,6 +117,13 @@ export class EOACode7702Transaction extends BaseTransaction<TransactionType.EOAC
     EIP2718.validateYParity(this)
     Legacy.validateHighS(this)
 
```

Comment:
I don't find this ban stated anywhere in the EIP.

Reply:
`validateHighS`? This is part of EIP-2 (Homestead), see rule 2 https://eips.ethereum.org/EIPS/eip-2

Reply:
I would assume the comment refers to the code part below (no `to` field), otherwise it would make no sense.

Reply:
(but I think this might be solved already during the chat discussion if I re-call my re-reading correctly?)

Reply:
Oh right, yes that would make more sense. Yes that's indeed resolved.